### PR TITLE
Slam 19 - amcl against static map

### PR DESCRIPTION
**Pull request by 1999michael**

### CODE REVIEW

**Comment by 1999michael** [comment]
So the changes that allowed AMCL against a static map have been added to move_base.launch, but didn't allow for us to tune the parameters. Will add an AMCL config file rather than reference husky_navigation one. 

Wiki page --> AMCL against static map

File: slam-ws/src/nav_husky_gazebo/config/global_costmap_params.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 global_costmap:
-  global_frame: /odom   # Because we don't have a map to load in, we use the odom frame as our global frame (rather than /map)
+  global_frame: /odom
```
Comment (erickmu1) [inline slam-ws/src/nav_husky_gazebo/config/global_costmap_params.yaml]: Keep this comment (maybe rephrase a bit) to explain why `odom` is used when no static map is supplied

File: slam-ws/src/nav_husky_gazebo/launch/move_base.launch [hunk 2]
```diff
@@ -1,20 +1,18 @@
+<?xml version="1.0"?>
 <launch>
 
    <master auto="start"/>
- <!-- Run the map server --> 
-    <!-- <node name="map_server" pkg="map_server" type="map_server" args="$(find my_map_package)/my_map.pgm my_map_resolution"/> -->
-
- <!--- Run AMCL --> 
-    <!-- <include file="$(find amcl)/examples/amcl_omni.launch" /> -->
```
Comment (erickmu1) [inline slam-ws/src/nav_husky_gazebo/launch/move_base.launch]: I believe these should be uncommented but instead use the `if` attribute w/in the `<node>` tag to specify whether these nodes should be run

File: slam-ws/src/nav_husky_gazebo/launch/move_base_amcl.launch [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
```
Comment (erickmu1) [inline slam-ws/src/nav_husky_gazebo/launch/move_base_amcl.launch]: Call `move_base.launch` from within this `.launch` file and specify non-default parameters/arguments when calling `move_base.launch` (so it can use the amcl config files)
